pkg/storage/storage.go

```diff
@@ -124,7 +147,7 @@ func (is *ImageStore) ValidateRepo(name string) (bool, error) {
 		return false, errors.ErrRepoNotFound
 	}
 
-	if len(files) != 4 {
+	if len(files) < 3 {
```

Comment:
I think I understand why s/!= 4/< 3/, but the comment at the top of this function should be updated to reflect this change, as it still says we expect exactly 4 entries.